pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ControllerPeriodicTasksIntegrationTests.java

```diff
@@ -386,7 +395,75 @@ public void afterRealtimeSegmentRelocatorTest(ITestContext context) throws Excep
     dropRealtimeTable(relocationTable);
   }
 
-  // TODO: tests for other ControllerPeriodicTasks (RetentionManager, BrokerValidationManager, OfflineSegmentIntervalChecker, RealtimeSegmentValidationManager)
+  @BeforeGroups(groups = "brokerResourceValidationManager")
+  public void beforeBrokerResourceValidationManagerTest(ITestContext context)
+      throws Exception {
+    String table1 = "testTable";
+    context.setAttribute("testTableOne", table1);
+    setupOfflineTable(table1);
+  }
+
+  @Test(groups = "brokerResourceValidationManager")
```

Comment:
Please add a dependsOn = "realtimeSegmentRelocator" to the before, test and after method. The idea is to make the tests run one after the other for each periodic task, incase any table setup/metrics is being shared.
Also, please add the new group in the comment right at the top, "The groups run sequentially in the order: segmentStatusChecker -> realtimeSegmentRelocation -> ...."

Reply:
Done.